src/pilot/templates/post-install.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+heat_template_version: 2014-10-16
+
+description: >
+  Example extra config for post-deployment
+
+parameters:
+  servers:
+    type: json
+
+resources:
+  ExtraDeployments:
+    type: OS::Heat::StructuredDeployments
+    properties:
+      servers:  {get_param: servers}
+      config: {get_resource: ExtraConfig}
+      # Do this on CREATE/UPDATE (which is actually the default)
+      actions: ['CREATE', 'UPDATE']
+
+  ExtraConfig:
+    type: OS::Heat::SoftwareConfig
+    properties:
+      group: script
+      config: |
+        #!/bin/bash
+
+        set -x
+        if [[ $HOSTNAME =~ "compute" ]] ; then
+          VCPUPINSET=$(crudini --get /root/parameters.ini DEFAULT novacpus)

```

Comment:
Both the NUMA/Hugepages patch and this one should set the vcpu_pin_set in the same way.  I like the tripleo method of doing it since there are fewer moving parts.

Reply:
I agree, and this is part of the future way of implementation. Both NUMA and Hugepages will be handled by former calsoft code, and we need now to think how we can eliminate this dependency interlock and put all of that together.